src/seedu/addressbook/Utils.java

```diff
@@ -0,0 +1,51 @@
+package seedu.addressbook;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * Utility methods
+ */
+public class Utils {
+
+    /**
+     * Throws an assertion error if any of the given arguments is null.
+     */
+    public static void assertNotNull(Object... items) {
+        int argIndex = 0;
+        for (Object item : items) {
+            if (Objects.isNull(item)) {
+                throw new AssertionError("Argument at index " + argIndex + " is null");
+            }
+            argIndex++;
+        }
+    }
```

Comment:
No need for the code to be defensive. i.e. don't verify assumptions at all. If we are going to do that, might as well use `assert` rather than write our own assert.